src/data/finset/image.lean

```diff
@@ -391,7 +393,7 @@ disjoint_iff_ne.2 $ λ a ha b hb, ne_of_apply_ne f $ h.forall_ne_finset
 
 lemma mem_range_iff_mem_finset_range_of_mod_eq' [decidable_eq α] {f : ℕ → α} {a : α} {n : ℕ}
   (hn : 0 < n) (h : ∀ i, f (i % n) = f i) :
-  a ∈ set.range f ↔ a ∈ (finset.range n).image (λi, f i) :=
+  a ∈ set.range f ↔ a ∈ (finset.range n).image f :=
```

Comment:
Can you revert the change to this line and the one below, to keep the diff small?
```suggestion
  a ∈ set.range f ↔ a ∈ (finset.range n).image (λi, f i):=
```